vignettes/pkgdown.Rmd

```diff
@@ -41,6 +41,8 @@ url: https://pkgdown.r-lib.org
 ```
 
 `url` is used throughout the site to generate absolute urls where they are needed.
+`url` is also part of what enables auto-links to your help topics or vignettes from sites external to your package, such as from other pkgdown sites or from Quarto websites.
```

Comment:
I think it's worth harping on this. I.e. that advertising the URL in all the right places is important for enabling incoming links from stuff you don't own.